tests/c/truncate.newcg.c

```diff
@@ -1,6 +1,6 @@
-// # Truncation currently always truncates to i32, no matter the target type.
-// #ignore-if: true
-// # ignore-if: test $YK_JIT_COMPILER != "yk" -o "$YKB_TRACER" = "swt"
+// ## Truncation currently always truncates to i32, no matter the target type.
```

Comment:
Are the `##`s intentional?

Reply:
Yep. See the last paragraph of the PR description.